pymc3/sampling.py

```diff
@@ -124,37 +124,26 @@ def sample(draws, step=None, start=None, trace=None, chain=0, njobs=1, tune=None
     step = assign_step_methods(model, step)
 
     if njobs is None:
+        import multiprocessing
         njobs = max(mp.cpu_count() - 2, 1)
```

Comment:
I wonder if we should just set the default to 2. That way everyone gets 2 chains unless they say otherwise, which allows Gelman-Rubin to be calculated.